docs/subset15.md

```diff
@@ -403,9 +404,14 @@ Two kinds of `import` are supported.
 
 * `list-constructor-expr` and `mapping-constructor-expr` are not allowed within a `const-expr`.
 * Types in type definitions are restricted semantically, rather than syntactically: a type definition that is referenced from a function definition must define a type that is equivalent to one that can be described using the type-defn grammar in this document. It must also match the type-defn [grammar supported for semantic type-checking](type-subset.md).
+* Structure types can not have subtypes of `function` as a member type.
```

Comment:
Put all these under a single bullet "Function typing is not yet fully implemented"

Reply:
Fixed